### DISCUSSION TITLE
Lerna-Lite changelog invalid character for Bug Fixes

### DISCUSSION THREAD

**Discussion by ghiscoding** [vuepress-theme-hope/vuepress-theme-hope#4191] [General]
hello @Mister-Hope I was looking at your changelog to see the Header request you asked to fix in Lerna-Lite and spotted a small error in your conventional changelog config

```ts
"section": "🪲 Bug Fixes"
```

resulting in an invalid symbol in your changelog (I assume that you may want to use 🐛 character). I also now see how you take advantage of the changelog `header`, it's pretty convenient to see the same changelog without the conventional-changelog commit guideline shown on top of your website... it's pretty cool!

Have a nice day

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/643976/4cc419a4-1281-4375-a2d2-43bcfc05be75)

**Comment by Mister-Hope** [comment]
👍 Thanks for catching this.

**Comment by Mister-Hope** [comment]
Hi, I still have some problems:



![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/9950b430-0a57-4419-8cdd-d09ead7642d6)

I am expecting the second comment being removed when releasing new versions, and I think I have the correct header set, but the previous comment is somehow preserverd? @ghiscoding

**Reply by ghiscoding** [reply]
No your `header` is set to ` "header": "# Changelog\n\nAll notable changes to this project will be documented in this file. See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.\n\n<!-- #region recent-change -->\n",` which is exactly what I'm seeing above, I'm sure that Lerna-Lite is ok now

**Reply by Mister-Hope** [reply]
yes, but look at the last line, the comment is not correctly removed in new release, 

![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/24137b7a-8492-438a-94fb-c1cdcb593897)

At least this is not something that I expect. See https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/59bd1429626850181a1be06b962b8a31854dcddf

I am expecting the whole header being recognized and the new content is inserted after it.

**Reply by ghiscoding** [reply]
Since you're now using `header`, then you moved the changelog header handling from Lerna-Lite to now conventional-changelog dependency, so it's not Lerna-Lite anymore. I don't know how they detect the header string replacement most probably RegEx (because that is how headers are handled in the changlog, it needs to read the changelog, finds the header remove it and re-add it on top every time you push a new release because that's the only way you append to a file) but I suspect that it's either your usage of special char `<!` or most probably the fact that you use a lot of empty lines (I wouldn't be surprised that anytime you use empty lines you end up with this problem, so you could try to remove the `\n` and see if that fixes it).

**Reply by Mister-Hope** [reply]
Got it, thanks for replying

**Reply by ghiscoding** [reply]
another thing to note is that because it does string replacement, if you change your header then the first time it runs it has problem to detect and erase that header. So if it's the first time you tried this header then it might be normal that it doesn't remove it completely but it works better the next time that you use/replace the same header
